doc/classes/AnimationNodeStateMachineTransition.xml

```diff
@@ -22,6 +22,9 @@
 		<member name="disabled" type="bool" setter="set_disabled" getter="is_disabled" default="false">
 			Don't use this transition during [method AnimationNodeStateMachinePlayback.travel] or [member auto_advance].
 		</member>
+		<member name="invert_condition" type="bool" setter="set_invert_condition" getter="get_invert_condition" default="false">
+			Turn on auto advance when [member advance_condition] is false
```

Comment:
Some stylistic changes to better follow the existing style:

```suggestion
			If [code]true[/code], disables automatic advance.
```

Also, I don't really know what "automatic advance" means in this context. It should probably be clarified.

Reply:
by auto advance, it refers to the "auto advance" property, which plays the next animation.
According to the class reference:

> Turn on the transition automatically when this state is reached. This works best with [constant SWITCH_MODE_AT_END].

Reply:
Thanks for clarifying! I updated the suggestion accordingly. Feel free to accept it as-is.

Reply:
That's a lot clearer! Thanks!